src/services/DidService.ts

```diff
@@ -36,23 +35,23 @@ export class DidService {
     } as Contact)
 
     const status = await did.store(myIdentity.identity)
-    if (status.type !== 'Finalised') {
+    if (status.type !== 'Finalized') {
       throw new Error(
         `Error creating DID for identity ${myIdentity.metaData.name}`
       )
     }
 
     persistentStore.store.dispatch(
       Wallet.Store.updateIdentityAction(myIdentity.identity.address, {
-        did: did.identifier,
+        did: { address: did.identifier, document: didDocument },
```

Comment:
Change request:
identifier (`kilt:did:abcd`) !== address (`abcd`), let's make this clear!

Reply:
I have changed this to Identifier everywhere. It was this before. 

What do you think @maudnals ?

Reply:
Great! Please make sure it doesn't break anything else